src/components/shared/form_elements/FormButton.vue

```diff
@@ -38,12 +39,17 @@ withDefaults( defineProps<Props>(), {
 	margin: 0;
 	font-size: 1em;
 	font-weight: 700;
+	text-align: center;
 	height: 54px;
+	line-height: 54px;
```

Comment:
![Bildschirmfoto vom 2024-06-05 16-17-27](https://github.com/wmde/fundraising-app-frontend/assets/11670701/de367acf-f8ca-49ec-8838-c8f748b83a2d)

this change affects other buttons as well, needs to be considered/changed

Reply:
Good catch, thanks!